cypress/page/iframe.page.ts

```diff
@@ -1,32 +1,26 @@
 
 class IframePage {
     private iFrameSite: string;
+    private iFrameSelector: string;
+    private cssSelector: string;
+
     
     constructor() {
       this.iFrameSite = "https://www.w3schools.com/html/html_iframe.asp";
+      this.iFrameSelector = "iframe[title*='Schools']";
+      this.cssSelector = "a[title^='CSS']"
     }
   
     public visit(): void {
       cy.visit(this.iFrameSite);
     }
   
     public getFrameTitle(htmlText: string): void {
-      cy.get('iframe').then($iframe => {
-        const $body = $iframe.contents().find('body') 
-        cy.wrap($body) 
-          .find('h1')
-          .should('have.text',htmlText)
-      })
+      cy.iframe(this.iFrameSelector).find('h1').should("have.text", htmlText);
```

Comment:
as the method is called getFrameTitle, I would change it to return the title, instead of performing the validation here:
`return cy.iframe(this.iFrameSelector).find('h1');`

And then perform the validation in the test class.